recipes-modules/wilc/wilc_15.5.bb

```diff
@@ -10,10 +10,11 @@ COMPATIBLE_MACHINE = "u96v2-sbc"
 SRC_URI = "git://github.com/linux4sam/linux-at91.git;protocol=git;branch=${BRANCH};subpath=drivers/net/wireless/microchip/wilc1000 \
            file://0001-ultra96-modifications-15.5.patch \
            "
+SRC_URI[sha256sum] = "a9c0d5259265632da32b17248cab6cd2fdee1e1ea06ee45407b90980e6b80296"
```

Comment:
I don't think we need this as we are cloning a repo and not downloading an archive

Reply:
This was an attempt to verify the patch file, which is what the sha256sum is for.  While not required (I just tested without that line and it builds fine) there is some security and safety associated with confirming the file matches the expected.  For an archived branch, it's also a nice confirmation that nothing has changed which could break things.

If including it is in conflict with a recommended practice for this repo, I can remove it.

Reply:
I also think it would be a good practice but from the yocto manual: If your [SRC_URI] statement includes URLs pointing to individual files fetched from **a remote server other than a version control system**, BitBake attempts to verify the files against checksums defined in your recipe to ensure they have not been tampered with or otherwise modified since the recipe was written.)
That's why I think it doesn't work with the github repository or the patch.
So for this recipe I would prefer not to have it.
Do you agree with this?

Reply:
Considering that Yocto also uses checksums to check local license files as a guiding practice, I am not sure that a remote (and public) version control system is the secure line they are trying to draw.  However, I understand the point made here, and if that is the best practice then I'll fall back to that.
I have removed the checksum line and pushed an update to my repo.  Let me know if there is anything else I need to update from the pull request.